purchase_batch_invoicing/wizards/purchase_batch_invoicing.py

```diff
@@ -101,9 +105,15 @@ def action_batch_invoice(self):
     @api.model
     def cron_invoice_all_pending(self, grouping="partner_id"):
         """Invoice all pending purchase orders."""
+        _logger.info("Starting to invoice all pending purchase orders.")
         wizard = self.create({
             "purchase_order_ids": self.env["purchase.order"].search(
                 self._purchase_order_domain()).ids,
             "grouping": grouping,
         })
-        return wizard.action_batch_invoice()
+        try:
+            wizard.action_batch_invoice()
+        except UserError as error:
+            _logger.warn(error.name)
```

Comment:
Yeah, but it's best to put it as info, and also move the other info to the try block. You get this way one of this option:
```
INFO: Starting to invoice all pending purchase orders.
INFO: Finished invoicing all pending purchase orders.
```
or
```
INFO: Starting to invoice all pending purchase orders.
INFO: No ready-to-invoice purchase orders selected.
```
I think this way is the best.